examples/simple-express-app/package.json

```diff
@@ -13,10 +13,7 @@
     "express": "^4.18.2",
     "serverless-http": "^3.2.0"
   },
-  "devDependencies": {
-    "serverless-dotenv-plugin": "^6.0.0"
-  },
```

Comment:
Pre-existing issue: uses a specific version instead of the version in the PR. This change along with the `npm link ../../` addition should resolve this.